lib/rules/trailing-semicolon.js

```diff
@@ -10,6 +10,21 @@ module.exports = {
   'detect': function (ast, parser) {
     var result = [];
 
+    var hasDoubleSemiColon = function (tree) {
+
+      tree.traverseByType('declarationDelimiter', function (node, index, parent) {
+        if (parent.content[index + 1].is('declarationDelimiter')) {
```

Comment:
Woud check that `parent.content[index + 1]` exists before calling `is`

Reply:
yup,